From 1a6526e6617fb014d695aa19bf80a3be5ba8994f Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Thu, 29 Sep 2022 19:15:03 +0000
Subject: [PATCH] GCC: remove redundant template parameter in
 content::FirstPartySetsHandler

GCC 12 in C++20 mode does not allow this anymore.
---
 content/public/browser/first_party_sets_handler.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/content/public/browser/first_party_sets_handler.h b/content/public/browser/first_party_sets_handler.h
index fbc86ae..9462aa9 100644
--- a/content/public/browser/first_party_sets_handler.h
+++ b/content/public/browser/first_party_sets_handler.h
@@ -51,12 +51,12 @@ class CONTENT_EXPORT FirstPartySetsHandler {
   template <typename T>
   class IssueWithMetadata {
    public:
-    IssueWithMetadata<T>(
+    IssueWithMetadata(
         const T& issue_type,
         const std::vector<absl::variant<int, std::string>>& issue_path)
         : issue_type_(issue_type), issue_path_(issue_path) {}
-    ~IssueWithMetadata<T>() = default;
-    IssueWithMetadata<T>(const IssueWithMetadata<T>&) = default;
+    ~IssueWithMetadata() = default;
+    IssueWithMetadata(const IssueWithMetadata<T>&) = default;
 
     bool operator==(const IssueWithMetadata<T>& other) const {
       return std::tie(issue_type_, issue_path_) ==
-- 
2.35.1

